jwst/pipeline/calwebb_ami3.py

```diff
@@ -110,29 +118,15 @@ def process(self, input):
 
                 # Perform blending of metadata for all inputs to this
                 # output file
-                self.log.info(
-                    'Blending metadata for averaged psf...'
-                )
-                self.log.info(
```

Comment:
Note sure why these log messages were here and of what use they were, so I got rid of them.